owncloudTestUtil/src/main/java/CustomViewMatchers.kt

```diff
@@ -0,0 +1,41 @@
+import android.view.View
+import android.view.ViewGroup
+import androidx.test.espresso.matcher.BoundedMatcher
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.hamcrest.Description
+
+fun withChildViewCount(count: Int, childMatcher: Matcher<View>): Matcher<View> {
```

Comment:
Feel free to move this utility to the ownCloudApp module as I suggested in the previous comment. We do already have a folder with custom matchers: https://github.com/owncloud/android/tree/master/owncloudApp/src/androidTest/java/com/owncloud/android/utils/matchers

Reply:
Remember to add the license header by the way 👍